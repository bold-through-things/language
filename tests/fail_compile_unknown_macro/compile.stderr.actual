[
  {
    "message": "Error at line 2: unknown macro: definitely_not_a_real_macro_name"
  }
]
