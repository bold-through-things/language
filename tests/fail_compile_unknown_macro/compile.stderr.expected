[
  {
    "recoverable": false,
    "line": 2,
    "char": 0,
    "content": "definitely_not_a_real_macro_name",
    "error": "unknown macro 'definitely_not_a_real_macro_name' - is this supposed to exist? did you maybe typo something?",
    "error_type": "INVALID_MACRO"
  }
]